README-CMake.md

```diff
@@ -61,9 +61,11 @@ cmake -DCMAKE_BUILD_TYPE=Release /path/to/klee/src
 
 * `ENABLE_ZLIB` (BOOLEAN) - Enable zlib support.
 
-* `GTEST_SRC_DIR` (STRING) - Path to GTest source tree.
+* `GTEST_SRC_DIR` (STRING) - Path to Google Test source tree. If it is not
```

Comment:
It is part of the llvm-source tree, right?

Reply:
It can be. But you might choose to use `llvm-config` (in which case you are not forced to reuse the CMake target) and/or just do not have the LLVM source tree available and instead specify a stand alone version of Google Test using this option. So there are plenty of use-cases for this option.

Or did you mean to point out something else?

Reply:
@jbuening Yes, I meant something else. The sentence finishes with `included within the LLVM build tree`, but its actually part of the source tree if I'm not mistaken.

Reply:
Yes, you are absolutely right, good catch! Fixed it.